airbyte-integrations/connectors/source-pinterest/source_pinterest/schemas/board_pins.json

```diff
@@ -9,6 +9,10 @@
     "created_at": {
       "type": ["null", "string"]
     },
+    "creative_type": {
+      "type": ["null", "string"],
+      "enum": ["REGULAR", "VIDEO", "CAROUSEL", "MAX_VIDEO", "SHOP_THE_PIN", "IDEA"]
+    },
```

Comment:
Just wondering, what was the symptom of tests failing as a result here? [There are many undocumented fields in the pinterest connector](https://github.com/airbytehq/airbyte/issues/24492), so I wouldn't expect tests to fail due to `fail_on_extra_columns: false`

Reply:
Tests failed because of expected and actual records mismatch

Reply:
Got it - i see quite a few more things in the diff
<img width="1040" alt="image" src="https://github.com/airbytehq/airbyte/assets/22333512/60629b1f-343f-4b7c-ba1a-8fa8ef8032e7">

But am fine with whatever additions make the test pass for now - I just feel like this wasn't the actual source of the mismatch 😄

Reply:
yep, that's just reordering - the only diff here is `"creative_type": "REGULAR"`